src/preamble.js

```diff
@@ -593,17 +593,16 @@ function abort(what) {
   }
 #endif
 
-  what += '';
+  what = 'Aborted(' + what + ')';
+  // TODO(sbc): Should we remove printing and leave it up to whoever
+  // catches the exception?
   err(what);
 
   ABORT = true;
   EXITSTATUS = 1;
 
 #if ASSERTIONS == 0
-  what = 'abort(' + what + '). Build with -s ASSERTIONS=1 for more info.';
-#else
-  var output = 'abort(' + what + ') at ' + stackTrace();
```

Comment:
I'm not sure why the stacktrace here was bring added the `what` message since the exception below will itself capture the stack trace.

Do we want to keep the `Build with -s ASSERTIONS=1 for more info.` in release builds?  Is that useful?

Reply:
Good question. It's hard to guess as to how many users benefit from this. I think we have docs that mention ASSERTIONS, but still, a message in the code is more direct...

How about this: let's remove it, but make sure that the FAQ has an entry mentioning that an `abort(..)` or any other exception thrown is a case where `-s ASSERTIONS` might help?

Reply:
I'll leave it for now to keep this change smaller, and we can followup with that perhaps.